docker/entrypoint-uwsgi.sh

```diff
@@ -27,7 +27,7 @@ python3 manage.py check
 
 DD_UWSGI_LOGFORMAT_DEFAULT='[pid: %(pid)|app: -|req: -/-] %(addr) (%(dd_user)) {%(vars) vars in %(pktsize) bytes} [%(ctime)] %(method) %(uri) => generated %(rsize) bytes in %(msecs) msecs (%(proto) %(status)) %(headers) headers in %(hsize) bytes (%(switches) switches on core %(core))'
 
-EXTRA_ARGS=""
+EXTRA_ARGS="${EXTRA_ARGS:-}"
```

Comment:
Isn't it better to call it `DD_UWSGI_EXTRA_ARGS`? As soon as `EXTRA_ARGS` appears in some configuration, it might be confusing: Extra args of what?

Reply:
+1